src/crimson/os/seastore/cache.cc

```diff
@@ -928,6 +928,7 @@ record_t Cache::prepare_record(Transaction &t)
   t.read_set.clear();
 
   record_t record;
+  record.commit_time = ceph::real_clock::now();
```

Comment:
I still recommend to use seastar/core/lowres_clock.hh according to previous discussions https://github.com/ceph/ceph/pull/26710#discussion_r268965531

> but can we use seastar::lowres_clock for better performance? ceph_clock_now() uses clock_gettime(CLOCK_REALTIME) under the hood. even it uses VDSO on modern Linux. it still takes an extra call, and presumably more expensive than reading a variable.

Reply:
sorry, missing a previous comment...

> It seems that lowres clock 's granularity is 10ms, I'm worried whether it's precise enough. I think maybe we keep ceph::real_clock for now, and change it if it proves to be a perf bottleneck in the future, what do you think?

I think 10ms is precise compared with the time from a segment written to be cleaned. Seems to me the replacement is not complicated, referring to https://github.com/ceph/ceph/pull/26710/commits/feddee4e5c5889a387a04d12dd61bd66b22085c9

What do you think?

Reply:
ok, will change it.